src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -139,12 +139,13 @@ internal void SetBuffer(byte[] buffer, int offset, int count)
         {
             _bufferedContent = new MemoryStream(buffer, offset, count, writable: false, publiclyVisible: true);
         }
-        
+
         internal bool TryGetBuffer(out ArraySegment<byte> buffer)
         {
+            buffer = default(ArraySegment<byte>);
```

Comment:
@ericstj had added that line when I was compiling a separate 'net46' OOB implementation against mscorlib, system. So, I left it in assuming it was needed.

But now that we are using a single OOB implementation and compiling against system.runtime, it isn't necessary. I'll remove it.